atomic_reactor/plugins/build_source_container.py

```diff
@@ -46,7 +46,6 @@ def export_image(self, image_output_dir: Path) -> str:
         img_metadata = get_exported_image_metadata(str(output_path), IMAGE_TYPE_DOCKER_ARCHIVE)
         # OSBS2 TBD exported_image_sequence will not work for multiple platform
         self.workflow.data.exported_image_sequence.append(img_metadata)
-        return str(output_path)
```

Comment:
we still want that

Reply:
@rcerven Could you be specific why we want this? The source docker archive can be accessible from the `BuildDir.exported_squashed_image`.

Reply:
right that will do